src/frontends/tensorflow_common/src/op/reciprocal.cpp

```diff
@@ -16,14 +21,41 @@ namespace op {
 
 OutputVector translate_reciprocal_op(const NodeContext& node) {
     // computes element-wise 1/x, where x - input
-    default_op_checks(node, 1, {"Reciprocal"});
+    default_op_checks(node, 1, {"Reciprocal"}, true);
     auto x = node.get_input(0);
-    auto minus_one_const = create_same_type_const_scalar<int32_t>(x, -1);
-    auto reciprocal = make_shared<v1::Power>(x, minus_one_const);
-    set_node_name(node.get_name(), reciprocal);
-    return {reciprocal};
+    auto complex_type_mark_x = as_type_ptr<ComplexTypeMark>(x.get_node_shared_ptr());
+    auto minus_one = make_shared<v0::Constant>(element::i32, Shape{1}, -1);
```

Comment:
please move `minus_one` under `if` branch since it is used only for complex type case.